src/controllers/costTime.controller.js

```diff
@@ -1,21 +1,26 @@
 'use strict'
 
-const { WASTE_IN_DEPOSIT_FOR_RECOVERY } = require('../constants').PermitTypes.STANDARD_RULES
+const Constants = require('../constants')
+const { WASTE_IN_DEPOSIT_FOR_RECOVERY } = Constants.PermitTypes.STANDARD_RULES
+const { MCP_CATEGORIES_PROCESSING_TIME, DEFAULT_CATEGORIES_PROCESSING_TIME } = Constants.PROCESSING_TIME
 const Routes = require('../routes')
 const BaseController = require('./base.controller')
 const CostTime = require('../models/taskList/costTime.task')
 const RecoveryService = require('../services/recovery.service')
+const StandardRuleType = require('../persistence/entities/standardRuleType.entity')
 
 module.exports = class CostTimeController extends BaseController {
   async doGet (request, h) {
     const pageContext = this.createPageContext(h)
     const context = await RecoveryService.createApplicationContext(h, { applicationLine: true, standardRule: true })
     const { applicationLine = {}, standardRule = {} } = context
+    const { categoryName } = await StandardRuleType.getById(context, standardRule.standardRuleTypeId)
 
     // Default to 0 when the balance hasn't been set
     const { value = 0 } = applicationLine
 
     pageContext.cost = value.toLocaleString()
+    pageContext.time = categoryName.includes('MCP') ? MCP_CATEGORIES_PROCESSING_TIME : DEFAULT_CATEGORIES_PROCESSING_TIME
```

Comment:
I'm not convinced that "contains the letters MCP" is the most robust check for these categories but I guess we haven't got anything more concrete to go on?